### PR TITLE
chg: filternode renderer to use base rendering again

### PR DESCRIPTION
**Pull request by Doralitze**
A few months ago I contributed a [change](https://github.com/pyqtgraph/pyqtgraph/pull/2869/files) to the pyqtgraph library that made it possible to change the rendering offsets of graph nodes. As this has been now packaged with a new release, we can finally get rid of a lot of redundant code.

### CODE REVIEW

File: src/view/logging_mode/logging_widget.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def new_log_message(self, message: str) -> None:
         """handle incoming log messages"""
         message_dict = json.loads(message)
         level = message_dict["level"]
+        if level == "WARNING":
+            level = "WARN"
```
Comment (Doralitze) [inline src/view/logging_mode/logging_widget.py:84]: This should not be required though.

Comment (maxkaussow) [reply src/view/logging_mode/logging_widget.py:84]: stimmt warum ist es da?

Comment (Doralitze) [reply src/view/logging_mode/logging_widget.py]: I just deleted this pice of code again. It turned out that the branch was out of date and did not include the logging fix that was already upstream.

**Review by maxkaussow — APPROVED** [review]
passt

**Pull request merged**